vunit/vivado/__init__.py

```diff
@@ -13,3 +13,5 @@
     add_from_compile_order_file,
     create_compile_order_file,
 )
+
+__all__ = ["run_vivado", "add_from_compile_order_file", "create_compile_order_file"]
```

Comment:
As far as I am aware, the purpose of `__all__` seems to be to reduce the number of imported elements when `from <module> import *` is used. However, doing so is generally considered a not good programming practice. In fact, we recently merged a PR to reduce the "catch them all" imports in VHDL (#992). Therefore, I'm not sure we want to support/enforce those kind of imports in Python.

Reply:
If I remember correctly: this will also make the documentation clearer, but I'll play around with it and see if it is actually the case.